libraries/api/src/index.js

```diff
@@ -132,6 +140,14 @@ class APIBuilder {
     if (options.scopes && !ScopeExpressionTemplate.validate(options.scopes)) {
       throw new Error(`Invalid scope expression template: ${JSON.stringify(options.scopes, null, 2)}`);
     }
+    if (options.implicitServiceScopes) {
+      let serviceScope = `service:${this.serviceName}:${options.name}`;
+      if (options.scopes) {
+        options.scopes = { AllOf: [options.scopes, serviceScope] };
```

Comment:
This would add additional required scopes, e.g., getting a secret would require both `secrets:get:<secret>` and `service:secrets:get`.  For already-scope-protected API endpoints, there's no need to add such additional scopes.

Reply:
That's a good point. Hand-tailored scope expressions are indeed better, and less magical, so the scope expressions are now all manually configured.